src/main/java/com/microsoft/sqlserver/jdbc/Util.java

```diff
@@ -40,6 +40,10 @@ static boolean isIBM() {
         return SYSTEM_JRE.startsWith("IBM");
     }
 
+    static String getJVMArch() {
+        return System.getProperty("os.arch").contains("64") ? "x64" : "x86";
```

Comment:
Is there no possibility that an OS is x64 and won't contain that information in its name? It wouldn't be too weird for certain Unix systems that've never supported 32-bit to not include it's bitness in the OS name. The following link seems to confirm that although unusual, it's possible: https://stackoverflow.com/questions/10846105/all-possible-values-os-arch-in-32bit-jre-and-in-64bit-jre/36926327. I'm personally in favor of the "load x64 bit dll first and load x86 if it errors" approach.

Reply:
- The property name is `os.arch`, but it returns JVM bitness. So if you install 32-bit JVM on 64-bit OS, the property will return `x86`.
- DLL will only be loaded on Windows, perhaps I can rename the method to `getJVMArchWindows()` and my tests show that the results are consistent(x86 for 32-bit, amd64 for 64-bit) for all Windows versions. See #907 for more details.
- We can't do "load x64 bit dll first and load x86 if it errors", because of https://github.com/microsoft/mssql-jdbc/blob/8d2c52ee049ca126ca9bcbdec4b8a0beb3096626/src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java#L6530.